### PR TITLE
fix: address goimports checks

### PR DESCRIPTION
**Pull request by sylr**
```
/home/sylvain/gopath/bin/goimports -w -l -srcdir ~/gopath/src/github.com/Azure/aks-engine main.go $(find . -type d ! -path ./vendor\* ! -path ./.git\* ! -path .)
```

### CODE REVIEW

**Comment by tariq1890** [comment]
@sylr Thanks for this PR. Since `goimports` has changed significantly in how it performs it's linting, we would need to discuss with the rest of the aks-engine members if we want to go forward with this approach. We'll  leave this PR open for the time being.

**Comment by mboersma** [comment]
@sylr thanks so much for taking the initiative to fix this! I think we've worked around it in #208 by updating `golangci-lint` instead, so I'm going to close this PR.

**Comment by tariq1890** [comment]
@sylr Can you rebase this ?

**Comment by sylr** [comment]
Ok

File: pkg/api/agentPoolOnlyApi/v20180331/apiloader_test.go [hunk 1]
```diff
@@ -5,7 +5,7 @@ package v20180331_test
 
 import (
 	"github.com/Azure/aks-engine/pkg/api"
-	"github.com/Azure/aks-engine/pkg/api/agentPoolOnlyApi/v20180331"
+	v20180331 "github.com/Azure/aks-engine/pkg/api/agentPoolOnlyApi/v20180331"
```
Comment (mboersma) [inline pkg/api/agentPoolOnlyApi/v20180331/apiloader_test.go]: The rest of the changes make sense, but I don't understand why this type of change is necessary. The `v20180331` package was already imported and used, so why do we need to pin it this way?

Comment (mboersma) [reply pkg/api/agentPoolOnlyApi/v20180331/apiloader_test.go]: We're assuming it's for readability. It makes `goimports` happy, so 🤷‍♂️.

**Pull request merged**